File: Bisq/src/main/java/com/joachimneumann/bisq/TransferPageAdapter.java
package com.joachimneumann.bisq;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentStatePagerAdapter;


public class TransferPageAdapter extends FragmentStatePagerAdapter {
    int mNumOfTabs;

    public TransferPageAdapter(FragmentManager fm, int NumOfTabs) {
        super(fm);
        this.mNumOfTabs = NumOfTabs;
    }

    @Override
    public Fragment getItem(int position) {

        switch (position) {
            case 0:
                TransferCodeQR tab1 = new TransferCodeQR();
                return tab1;
            case 1:
                TransferCodeEmail tab2 = new TransferCodeEmail();
                return tab2;
            default:
                return null;
        }
    }

    @Override
    public int getCount() {
        return mNumOfTabs;
    }
}

File: Bisq/src/main/java/com/joachimneumann/bisq/TransferPageAdapter.kt
package com.joachimneumann.bisq

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter


class TransferPageAdapter(fm: FragmentManager, internal var mNumOfTabs: Int, internal var message: String) : FragmentStatePagerAdapter(fm) {

    override fun getItem(position: Int): Fragment? {
        val bundle = Bundle()
        bundle.putString("bisqPhoneID", message)
        when (position) {
            0 -> {
                val tab1 = TransferCodeQR()
                tab1.arguments = bundle
                return tab1
            }
            1 -> {
                val tab2 = TransferCodeEmail()
                tab2.arguments = bundle
                return tab2
            }
            else -> return null
        }
    }

    override fun getCount(): Int {
        return mNumOfTabs
    }
}